--- extconf.rb.org	Thu Jun 20 18:11:45 2013
+++ extconf.rb	Thu Jun 20 18:12:07 2013
@@ -44,8 +44,8 @@
   have_library("gdi32")
 end
 result = have_header("openssl/ssl.h")
-result &&= %w[crypto libeay32].any? {|lib| have_library(lib, "OpenSSL_add_all_digests")}
-result &&= %w[ssl ssleay32].any? {|lib| have_library(lib, "SSL_library_init")}
+result &&= %w[crypto].any? {|lib| have_library(lib, "OpenSSL_add_all_digests")}
+result &&= %w[eissl].any? {|lib| have_library(lib, "SSL_library_init")}
 if !result
   unless pkg_config("openssl") and have_header("openssl/ssl.h")
     message "=== Checking for required stuff failed. ===\n"
